src/Apps/Collect2/Routes/Collection/Components/Header/index.tsx

```diff
@@ -180,11 +180,13 @@ export const CollectionHeader: FC<Props> = ({ artworks, collection }) => {
       {({ xs, sm, md, lg }) => {
         const size = xs ? "xs" : sm ? "sm" : md ? "md" : lg ? "lg" : "xl"
         const imageWidth = imageWidthSizes[size]
-        const imageHeight = xs ? imageHeightSizes.xs : imageHeightSizes.sm
+        const smallerScreen = size === "xs" || size === "sm"
+        const imageHeight = smallerScreen
+          ? imageHeightSizes.xs
+          : imageHeightSizes.sm
         const chars = maxChars[size]
         const categoryTarget = `/collections#${slugify(collection.category)}`
         const artistsCount = size === "xs" ? 9 : 12
```

Comment:
I know this isn't part of the work you were doing with this PR, but this looks like another place we could use the booleans passed into the function (`xs`, ...) instead of a string comparison.